src/tasks/lm_parsing.py

```diff
@@ -0,0 +1,150 @@
+"""
+Task definitions for language modeling parsing tasks. Used for unsupervised parsing encoders.
+Long term dependencies for language modeling: sentences concatenated together seperated by <EOS> token.
+"""
+import json
+import logging as log
+import math
+import os
+
+from allennlp.training.metrics import Average
+from allennlp.data.token_indexers import SingleIdTokenIndexer
+
+# Fields for instance processing
+from allennlp.data import Instance, Token
+
+from ..utils.utils import process_sentence, truncate, load_tsv
+
+from typing import Iterable, Sequence, List, Dict, Any, Type
+
+from .tasks import SequenceGenerationTask
+from .tasks import sentence_to_text_field, atomic_tokenize
+from .tasks import UNK_TOK_ALLENNLP, UNK_TOK_ATOMIC
+from .lm import LanguageModelingTask
+from .registry import register_task
+
+
+class LanguageModelingParsingTask(LanguageModelingTask):
+    def count_examples(self):
+        """Computes number of samples.
+        Every example is made up of sentences concatenated together, capped by max_seq_len.
+        """
+        example_counts = {}
+        for split, split_path in self.files_by_split.items():
+            arr = [line.strip().split()+["<EOS>"] for line in open(split_path)]
```

Comment:
autopep8!

Reply:
I ran autopep8! It fixed a few issues, but not this line apparently.

Reply:
Odd. Here's my usual invocation:

python -m autopep8 --max-line-length 100  --in-place --aggressive  --aggressive  ./*.py ./*/*.py ./*/*/*.py